entity_matching/README.md

```diff
@@ -0,0 +1,16 @@
+To run the notebook locally install poetry with https://github.com/python-poetry/poetry, then:
```

Comment:
Let's skip the local development readme for now maybe? Instead, we should have a readme that explains what this is.